src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheck.java

```diff
@@ -163,6 +163,10 @@
  * }
  * </pre>
  * <p>
+ * For all the above examples, the handling of 'record' introduced in Java 14 and 'class'
+ * is the same.
```

Comment:
this sounds strange ... as it is also applicable to interfaces and all other target tokens.